src/models/network.js

```diff
@@ -41,6 +42,10 @@ function Network(attr) {
 		chanCache: [],
 	});
 
+	if (!this.uuid) {
```

Comment:
Why do we need to check for previous ids?

Reply:
I didn't want to sheets to always call uuid() in default object as it would waste the call when loading an existing network.